### PR TITLE
refactor: migratation to jina3

### PR DESCRIPTION
**Pull request by samsja**
DO NOT MERGE before 
* merging https://github.com/jina-ai/executor-text-clip-encoder/pull/5 into main
* changing back the cicd to point to the master branch of the workflow-executors repo 
* changing the dockerfilegpu to use the last official jina release

### CODE REVIEW

**Comment by samsja** [comment]
DO NOT MERGE before 
* merging https://github.com/jina-ai/executor-text-clip-encoder/pull/5 into main
* changing back the cicd to point to the master branch of the workflow-executors repo

File: main.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from jina import Flow
+
+from clip_text import CLIPTextEncoder
```
Comment (JoanFM) [inline main.py]: why do u add a main? pls remove

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

**Comment by samsja** [comment]
https://github.com/jina-ai/internal-tasks/issues/379

**Comment by mapleeit** [comment]
Seems we should git ignore `.idea` and `__pycache__`?

**Pull request merged**